modules/jsonld/src/test/scala/ch/epfl/bluebrain/nexus/rdf/jsonld/JsonLdSpec.scala

```diff
@@ -161,6 +167,57 @@ class JsonLdSpec extends RdfSpec {
           .value
           .rightValue shouldEqual expected
       }
+
+      "there are scoped contexts" in {
```

Comment:
Can we add to the json some other values to see that they don't get removed like the context does:

```
{
  "@context": {
    "xsd": "http://www.w3.org/2001/XMLSchema#"
  },
  "key": {
    "@context": {
      "xsd": "http://www.key.com#"
    },
   "key-value1": "value1", 
    "key2": {
      "@context": {
        "schema": "http://schema.org/"
      },
   "key-value2": "value2", 
    }
  }
}
```